src/__template__/ComponentTemplate.vue

```diff
@@ -56,6 +56,15 @@ const props = defineProps({
     required: false,
     default: 'span',
   },
+  /**
+   * A boolean to enable prefixing any internal SVG ids with a unique string; useful when there are potentially multiple SVG instances on the same page and the SVG utilizes ids and references internally (e.g. in the `<defs>` tag).
+   * Typically only set to false during snapshot testing.
+   * Defaults to `true`.
+   */
+  randomizeIds: {
+    type: Boolean,
+    default: true,
+  },
```

Comment:
The prop is really only in place to standardize the ids for the snapshot testing. 

Since the attributes are nested (not just a JS object) I can't simply assert that they evaluate to "any string" in Vitest 😢.

This prop shouldn't be disabled other than for snapshot testing; so defaulting to `true` as-is seems ideal? 

I actually don't think I could switch to `useId()` regardless since I need to generate a unique ID per selector instance at runtime, meaning I wouldn't be able to call `useId()` at the root of the setup function without first calculating how many ids I need total and then calling `useId()` once per instance. Either way, the ids would _still_ need to be randomized at runtime.

Reply:
> Can we reverse the semantics and use something like `stableIds` or `fixedIds`? So that 1. the default value can be `false`

Rethinking this; yes I'll rename and flip to be `false`